CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to
 ### Added
 
 - Add synchronization for course runs
+- Add make psql cmd to access postresql's cli in docker
```

Comment:
Not sure about the reference to Docker since everything is in Docker anyway:
```suggestion
- Add make psql cmd to access postgresql's cli
```

Same in your commit message:
`s/access postresql docker with make psql/access postgresql with make psql`